extensions/sendgrid/v1/actions/sendEmailWithTemplate/config/fields.ts

```diff
@@ -0,0 +1,76 @@
+import { isNil, isEmpty } from 'lodash'
+import { z, type ZodTypeAny } from 'zod'
+import {
+  type Field,
+  FieldType,
+  StringType,
+} from '@awell-health/extensions-core'
+
+export const fields = {
```

Comment:
Let's add fromName and fromEmail here as well so it can be overwritten from the extension settings if needed. They can both be optional and use the following description:

fromName: The name that will be used for the "From" header. When left blank, the value specified in the extension settings will be used.
fromEmail: The email address that will be used for the "From" header. When left blank, the value specified in the extension settings will be used.
The logic should be as follows:

When fromName or fromEmail is defined on the action level, then that value takes precedence over the value defined in the settings
When fromName or fromEmail is NOT defined on the action level, then we use the value from the settings.
When fromName or fromEmail is NOT defined on the action level and NOT defined in the settings = throw a validation error